redis/src/cluster.rs

```diff
@@ -96,11 +96,11 @@ impl TlsMode {
 
 impl ClusterConnection {
     pub(crate) fn new(
-        cluster_params: &ClusterParams,
-        initial_nodes: &[ConnectionInfo],
+        cluster_params: ClusterParams,
```

Comment:
These changes should be part of the "use ClusterParams struct to pass params" commit.